identity/app/views/signin.scala.html

```diff
@@ -16,6 +17,8 @@
     <form class="form js-signin-form" novalidate action="@idUrlBuilder.buildUrl("/signin", idRequest)" role="main" method="post">
         @if(signinForm.globalError.isDefined) {
             <div class="form__error">@signinForm.globalErrors.map(_.message).mkString(", ")</div>
+        } else {
```

Comment:
This error should be only on the registration page.  We agreed yesterday that any failures to sign-in with Facebook are in effect registrations.  I.e. the user is re-registering.  Therefore, the social app will always redirect to the registration page given email missing in the response.  Also the error text doesn't quiet make sense on the sign-in page.